test/unit/transaction-runner-test.coffee

```diff
@@ -96,7 +96,7 @@ describe 'TransactionRunner', ->
 
         assert.isOk runner.multiBlueprint
 
-  describe 'configureTransaction(transaction, callback)', ->
+  describe 'configureTransaction(transaction)', ->
```

Comment:
This is unrelated change, see https://github.com/apiaryio/dredd/pull/906/files#r152850808